Formula/mavsdk.rb

```diff
@@ -4,8 +4,8 @@ class Mavsdk < Formula
   desc "API and library for MAVLink compatible systems written in C++17"
   homepage "https://mavsdk.mavlink.io"
   url "https://github.com/mavlink/MAVSDK.git",
-      tag:      "v1.4.12",
-      revision: "1e08fccd593b25970b87df9f2609ce4907ceae1f"
```

Comment:
this wont work. it needs `version_scheme 1`

Reply:
Can we also have some upstream issue to understand why the tag got removed?

Reply:
OK, I opened https://github.com/mavlink/MAVSDK/issues/1978. Should `version_scheme 1` be added now, or should we wait for upstream's response?

Reply:
let's wait a bit for the upstream feedback on the tag thing first :)

Reply:
anyway, I saw you already updated it, should be good now. (that also works!)